templates/homepage.html

```diff
@@ -34,11 +34,23 @@
 </head>
 
 <body>
+    <div id="user_info">
+        <a href="/login">Login</a>
+        <br>
```

Comment:
Rather than adding `<br>` tags, you can use css to target the anchor tags in this div and give them margin